### PR TITLE
Test for activating the same cache path

### PR DESCRIPTION
**Pull request by pskwarlo**

### CODE REVIEW

**Comment by Deixx** [comment]
ok to test

**Comment by Deixx** [comment]
retest this please

File: test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import pytest
+
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Size, Unit
+
+cache_size = Size(16, Unit.GibiByte)
+block_size = Size(1, Unit.Blocks512)
```
Comment (mmichal10) [inline test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py:14]: Those variables seems redundant

Comment (pskwarlo) [reply test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py]: removed

**Review by mmichal10 — DISMISSED** [review]

File: test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        cache.stop()
+
+    with TestRun.step("Load standby cache on the partition with valid CAS metadata."):
+        cache = casadm.standby_load(cache_dev)
+
+    with TestRun.step("Detach standby cache"):
+        cache.standby_detach()
+
+    with TestRun.step("Activate cache providing path to the partition."):
+        cache.standby_activate(device=cache_dev)
```
Comment (pdebski21) [inline test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py:33]: Consider adding check to determine if activate succeeded.
You can use cache statistics or check if exported object appeared properly.

File: test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_standby_activate_the_same_cache_path():
```
Comment (pdebski21) [inline test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py:16]: Add test description and pass criteria.
It's most offten in that form. Check also other tests for that.
    
```
    """
    title: <title>
    description: |
        <desc>
    pass_criteria:
      - <crit>
      - <crit>
    """
```

File: test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py (hunk 2 above)
Comment (pskwarlo) [reply test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py:45]: done

File: test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py (hunk 3 above)
Comment (pskwarlo) [reply test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py:18]: done

**Review by mmichal10 — APPROVED** [review]

**Review by pdebski21 — APPROVED** [review]

**Pull request merged**